### ISSUE TITLE
[Enhancement] - Show new Cloud Events at top

### ISSUE THREAD

**Issue by lamw** [n3wscott/sockeye#16]
When Sockeye is connected up to a vSphere-based environment, the number of events can be quite chatty (almost 100+ in few minutes) and can be difficult to quickly find the event you're interested in. Often times, I need to scroll all the way down and then slowly scroll up to search for the event before I can apply the filter to then reduce the noise.

I think this would be a good UX improvement

**Comment by JeffNeff** [comment]
I agree @lamw .. I don't think it makes much sense for new events to appear at the bottom.. let's fix this!

**Comment by n3wscott** [comment]
I did try this and the rendering did not make a lot of sense, the lock button at the bottom will keep you at the newest event.
